client/src/components/ui/Card/Card.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC, PropsWithChildren } from 'react';
+import classes from './Card.module.scss';
+
+type CardProps = {
+	height?: string;
+	width?: string;
+};
+
+export const Card: FC<PropsWithChildren<CardProps>> = ({
+	children,
+	height,
+	width,
+}) => {
```

Comment:
remove props, card will only be a wrapper and sizing will be handled by children